### PR TITLE
feat(test): Added more tests

### PR DESCRIPTION
**Pull request by joaorufi**
**Proposed Changes**
- Added more tests

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: pkg/report/asff_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		t.Run(test.name, func(t *testing.T) {
+			if err := os.MkdirAll(test.caseTest.path, os.ModePerm); err != nil {
+				t.Fatal(err)
+			}
+
+			err := PrintASFFReport(test.caseTest.path, test.caseTest.filename, test.caseTest.summary)
+			require.NoError(t, err)
+
+			require.FileExists(t, filepath.Join(test.caseTest.path, "asff-"+test.caseTest.filename+".json"))
+		})
```
Comment (rafaela-soares) [inline pkg/report/asff_test.go:37]: Maybe we should remove the file

```suggestion
			require.FileExists(t, filepath.Join(test.caseTest.path, "asff-"+test.caseTest.filename+".json"))
			os.RemoveAll(test.caseTest.path)
		})
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: pkg/report/asff_test.go (hunk 1 above)
Comment (joaorufi) [reply pkg/report/asff_test.go:37]: I'm using `/tmp` dir to save files, so I think there is no need to delete them, since they are deleted automatically

Comment (rafaela-soares) [reply pkg/report/asff_test.go:37]: Are you sure? At least, in Windows, my files in `/tmp` seem to not be deleted automatically

**Review by joaoReigota1 — APPROVED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM

**Pull request merged**